languageserver/integration/templates.go

```diff
@@ -0,0 +1,83 @@
+package integration
+
+
+const deployContractTemplate = `
+transaction(name: String, code: [UInt8]) {
+  prepare(signer: AuthAccount) {
+    if signer.contracts.get(name: name) == nil {
+      signer.contracts.add(name: name, code: code)
+    } else {
+      signer.contracts.update__experimental(name: name, code: code)
+    }
+  }
+}
+`
+
+const contractAccountManager = `
```

Comment:
What is this account manager used for?

Reply:
The idea was to store created account on chain, so Language Server could pick up/resolve name to address from the chain.
Since language server can "easily" crash loosing all it's state in the process.
With this approach it's even possible to restore crashed VS Code/extension without restarting emulator and losing it's state :)
Also you will get consistent name/address mapping every time you work with the project, since created accounts get assigned name in order.

I can illustrate the idea better during the demo.

Reply:
Ah, neat!